src/content/en/2020/http2.md

```diff
@@ -285,14 +350,16 @@ Figure 22.14 shows the breakdown of HTTP/2 responses for 3rd party CDNs for mobi
 <figcaption>{{ figure_link(caption="Percentage of HTTP/2 requests served by the third-party resouce CDNs over mobile", sheets_gid="781660433", sql_file="cdn_detail_by_cdn.sql") }}</figcaption>
 </figure>
 
-## How is HTTP/2 performing ?
+{# TODO(authors, editors): Sort of a nit. Headings inconsistently ask questions and give short titles to describe the contents. It would be more natural to have a unified naming convention for headers. #}
+## How is HTTP/2 performing?
 
-Measuring the impact of how a protocol is performing is difficult with the current HTTPArchive approach. It would be really fascinating to be able to quantify the impact of concurrent connections, the effect of packet loss and differrent congestion control mechanisms. To really compare performance each web site would have to be crawled over each protocol over different network conditions. What we can do instead is to look into the impact on the number of connections a web site uses.
+Measuring the impact of how a protocol is performing is difficult with the current HTTP Archive [approach](./methodology). It would be really fascinating to be able to quantify the impact of concurrent connections, the effect of packet loss, and differrent congestion control mechanisms. To really compare performance, each website would have to be crawled over each protocol over different network conditions. What we can do instead is to look into the impact on the number of connections a website uses.
 
-### Reducing connectons
+### Reducing connections
 
-As discussed earlier, HTTP/1.1 only allowed a single request at a time over a TCP connection. Most browsers got around this by allowing 6 parallel connections per host. The major improvement with HTTP/2 is that multiple requests can be multiplexed over a single TCP connection.  This should reduce the total number of connections, and the associated time and resources required, to load a page.
+As discussed [earlier](#http10-to-http2), HTTP/1.1 only allows a single request at a time over a TCP connection. Most browsers get around this by allowing six parallel connections per host. The major improvement with HTTP/2 is that multiple requests can be multiplexed over a single TCP connection.  This should reduce the total number of connections—and the associated time and resources—required to load a page.
 
+{# TODO(analysts): Nit, but I think it'd make more sense to show 2020 to the right of 2016 as I think most readers will conceptualize time going from left to right. #}
```

Comment:
@bazzadp

Reply:
Fair point. Fixed the graphs affected and added fall back images in https://github.com/HTTPArchive/almanac.httparchive.org/pull/1618/commits/d825f2b5a14babe878e1f3a087cc7ecce159994d